benchmark/internal/cireport/metareport_test.go

```diff
@@ -3,9 +3,8 @@ package cireport_test
 import (
 	"github.com/pyroscope-io/pyroscope/benchmark/internal/cireport"
 
-	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-	"github.com/tommy351/goldga"
```

Comment:
need to remove this because this package still use v1, will lead to panic
https://github.com/onsi/ginkgo/issues/875

Reply:
Not great, but not a big deal in my opinion. What do you think @eh-am ?

Reply:
@dhanusaputra Can you try this fork I just created? https://github.com/pyroscope-io/goldga I migrated goldga library to use ginkgo v2

Reply:
thanks for help
looks ok